w3-blob.md

```diff
@@ -418,7 +472,7 @@ Receipt MUST not have any effects.
 
 ## Accept Blob
 
-Authorized agent MAY invoke `/service/blob/accept` capability on the [provider] subject. Invocation MUST either succeed when content is delivered on allocated `address` or fail if either allocation failed or expired before content was delivered.
+Authorized agent MAY invoke `/service/blob/accept` capability on the [provider] subject. Invocation MUST either succeed when content is delivered to an allocated `address` or fail if either allocation failed or expired before content was delivered.
 
 Invocation MUST block until content is delivered. Implementation MAY resume when content is sent to the allocated address or await until client signals that content has been delivered using [put blob receipt].
```

Comment:
also looking at the first receipt, I think we should have:

```typescript
type BlobAccept = {
  cmd: "/service/blob/accept"
  sub: ProviderDID
  args: {
    blob: Blob
    exp: int
    space: SpaceDID
    _put: [".out.ok.site", Link<HttpPut>]
  }
}
```

is this correct?

Reply:
actually on `BlobAccept` we have an issue at the moment: 
1. I agree that we need to have `space` as arg, but we have no way to find out the space, unless we pass it via `http/put`. What do you think @Gozala ?

Reply:
Also, `blobAccept` handler needs to do set in Assert.location agent DID as audience, not space right?

Reply:
No it should be a space, because agent acts on space authority. E.g agent may be CI job authorized to upload content, but not authorized to publish it on e.g bitswap

Reply:
ok, that seems correct. so then question remains to, how can we get it unless we pass it over on `http/put`

Reply:
We talked about this in sync and have concluded that we should not capture `space` in the `http/put` because technically there could be multiple `blob/accept`'s across several spaces awaiting for the blob content and it does not matter which one will deliver it. In fact `space` where content is delivered is web3.storage space.

However completing `blob/accept` on other all spaces awaiting is out of scope for the current implementation. That said we can still obtain the space info from the `/http/put` because it's `url` and `headers` args will have `ucan/await` linked to the `/service/blob/allocate` that has `space` argument.

Reply:
thanks for summarizing here. 

@Gozala I think the first commend in this thread is still relevant though:

> can't suggest there because is not part of the diff but `blob/accept` receipt type should have site right?

```typescript
type BlobAcceptOk = {
  site: Link<LocationClaim>
}
```